src/lib/profiles/data-management/Current/UpdateClient.h

```diff
@@ -122,7 +122,7 @@ struct UpdateClient::InEventParam
 struct UpdateClient::OutEventParam
 {
     bool DefaultHandlerCalled;
-    union
+    union _ // make clang 9 happy. remove name when adding members.
```

Comment:
Could we instead add an empty member struct for one of the API events?

```
union
{
    struct
    {
    } UpdateComplete; // Empty structure to avoid gratuitous clang warning about unions without members.
};
```
Will this not satisfy the compiler?

Reply:
Yes, this compiles.
Just a reminder, an empty struct's size in C++ is not 0. Let's apply this one if it does not break anything.